src/data/finset/card.lean

```diff
@@ -308,6 +308,11 @@ lemma card_sdiff (h : s ⊆ t) : card (t \ s) = t.card - s.card :=
 suffices card (t \ s) = card ((t \ s) ∪ s) - s.card, by rwa sdiff_union_of_subset h at this,
 by rw [card_disjoint_union sdiff_disjoint, add_tsub_cancel_right]
 
+lemma card_sdiff_ge (s t : finset α)  : card (t \ s) ≥ t.card - s.card :=
```

Comment:
no ge (the linter will yell at you)

Reply:
Hmm, I thought it's more natural to put the operator of interest to the left, but I understand that it's preferable to always use `le`. Will fix.

Reply:
What should the name be now? Still `card_sdiff_ge`? Or `card_sdiff_le` (good to discover next to `card_diff`)?
Or something like `card_sub_card_le_card_dsiff`?

Reply:
@digama0, can you advise here about the name? I am reluctant to just merge lest I get the name wrong.

Reply:
`card_sub_le_card_dsiff` seems reasonable to me, or just `le_card_dsiff`